ext/nnstreamer/tensor_decoder/tensordec-flatbuf.cc

```diff
@@ -0,0 +1,170 @@
+/**
+ * GStreamer / NNStreamer tensor_decoder subplugin, "Flatbuffer"
+ * Copyright (C) 2020 Gichan Jang <gichan2.jang@samsung.com>
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Library General Public
+ * License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Library General Public License for more details.
+ *
+ */
+/**
+ * @file        tensordec-flatbuf.cc
+ * @date        26 Feb 2020
+ * @brief       NNStreamer tensor-decoder subplugin, "flatbuffer",
+ *              which converts tensor or tensors to flatbuffer byte stream.
+ *
+ * @see         https://github.com/nnsuite/nnstreamer
+ * @author      Gichan Jang <gichan2.jang@samsung.com>
+ * @bug         No known bugs except for NYI items
+ *
+ */
+
+#include <iostream>
+#include <typeinfo>
+#include <glib.h>
+#include <gst/gstinfo.h>
+#include <nnstreamer_plugin_api_decoder.h>
+#include <nnstreamer_plugin_api.h>
+#include <../include/nnstreamer_generated.h> // Generated by `flatc`.
```

Comment:
Let's place internal & generated header more "private" location (refer to tensor-transform)

Reply:
generated file is moved to build dir.